Cargo.toml

```
@@ -10,24 +10,24 @@ description = "Cross-platform WebView rendering library"
 readme = "README.md"
 repository = "https://github.com/tauri-apps/wry"
 documentation = "https://docs.rs/wry"
-categories = [ "gui" ]
-exclude = [ "/.changes", "/.github", "/audits", "/wry-logo.svg" ]
+categories = ["gui"]
+exclude = ["/.changes", "/.github", "/audits", "/wry-logo.svg"]
 
 [package.metadata.docs.rs]
 no-default-features = true
-features = [ "drag-drop", "protocol", "os-webview" ]
+features = ["drag-drop", "protocol", "os-webview"]
 targets = [
   "x86_64-unknown-linux-gnu",
   "x86_64-pc-windows-msvc",
-  "x86_64-apple-darwin"
+  "x86_64-apple-darwin",
 ]
-rustc-args = [ "--cfg", "docsrs" ]
-rustdoc-args = [ "--cfg", "docsrs" ]
+rustc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
 default = [ "drag-drop", "objc-exception", "protocol", "os-webview" ]
 serde = [ "dpi/serde" ]
-objc-exception = [ "objc/exception" ]
+objc-exception = ["objc2/exception"]
```

I read the docs in `tauri` today, what you actually want is the `"catch-all"` feature:

```suggestion
objc-exception = ["objc2/catch-all"]
```